client/src/pages/CreateBrofile.js

```diff
@@ -0,0 +1,14 @@
+import { useState } from 'react';
+import Form from '../components/Form';
+
+export default function CreateBrofile({ open }) {
+    const [brofile, setBrofile] = useState([]);
+
+
+    return (
+        <>
+            <Form open={open} ></Form>
+        </>
+    )
```

Comment:
```suggestion
    return <Form open={open} ></Form>
```
I think you don't need the Fragments and Brackets as you only render one component.